app/liquid_tags/codeit_tag.rb

```diff
@@ -0,0 +1,28 @@
+class CodeitTag < LiquidTagBase
+  def initialize(tag_name, id, tokens)
+    super
+    @url = parse_url(url)
+  end
+
+  def render(_context)
+    '<iframe src="' + @url + '"
+      scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" loading="lazy" style="width: 100%;" height="600"
+    </iframe>'
+  end
+
+  private
+
+  def parse_url(url)
+    if (url.match(/^https:\/\/cde.run\//) || url.match(/^https:\/\/dev.cde.run\//))
+      url = url
+    else
+      url = 'https://cde.run/' + url
+    return url
+    end
+  end
+
+  end
+end
```

Comment:
Thanks so much for the PR @barhatsor ! It looks like you have an extra `end` in here. The extra leading spaces on line 24 make it hard to spot, but that's what's causing the test failures in CI.